### PR TITLE
Add richardhj extensions

### PR DESCRIPTION
**Pull request by richardhj**

### CODE REVIEW

File: linter/whitelists/default.txt [hunk 1]
```diff
@@ -127,6 +129,7 @@ PNG
 Podcasts
 POIs
 QR
+Relaod
```
Comment (fritzmg) [inline linter/whitelists/default.txt]: Typo :)

File: meta/richardhj/contao-ajax_reload_element/de.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+de:
+    title: Ajax Reload Element
+    description: >
+        Diese Erweiterung bietet die Möglichkeit, ein bestimmtes Frontend-Modul oder Inhaltselement über einen Ajax-Request zu laden.
+
+        Nach der Installation können Sie jedes Element, das über Ajax geladen werden darf, markieren. Danach kannst du dieses Element über den bereitgestellten Ajax-Endpunkt laden.
```
Comment (fritzmg) [inline meta/richardhj/contao-ajax_reload_element/de.yml]: Mixed "Sie" & "du"

File: meta/richardhj/contao-ajax_reload_element/en.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+en:
+    title: Ajax Reload Element
+    description: >
+        Ajax Reload Element gives you the possibility to fetch a particular front end module or content element via an ajax request.
```
Comment (xchs) [inline meta/richardhj/contao-ajax_reload_element/en.yml]:
```suggestion
        Ajax Reload Element gives you the possibility to fetch a particular front end module or content element via an Ajax request.
```

File: meta/richardhj/contao-ajax_reload_element/en.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+        After installation, you can specify any element that is allowed to fetch via Ajax with a checkbox. As a result, you are to fetch this element using the provided Ajax-Endpoint.
```
Comment (xchs) [inline meta/richardhj/contao-ajax_reload_element/en.yml]:
```suggestion
        After installation, you can specify any element that is allowed to fetch via Ajax with a checkbox. As a result, you are to fetch this element using the provided Ajax endpoint.
```

File: meta/richardhj/contao-ajax_reload_element/en.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+        This extension comes with a jQuery-template to automatically transform any Contao form to an Ajax form.
```
Comment (xchs) [inline meta/richardhj/contao-ajax_reload_element/en.yml]:
```suggestion
        This extension comes with a jQuery template to automatically transform any Contao form to an Ajax form.
```

File: meta/richardhj/contao-email-token-login/de.yml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+de:
+    title: Token-Login
+    description: >
+        Diese Erweiterung bietet ein alternatives Anmeldeformular, für das die Mitglieder kein Passwort benötigen.
+
+        Das Anmeldeformular fragt nur nach dem Benutzernamen (oder der E-Mail-Adresse). Die Benutzer*in sendet das Formular ab und erhält eine Benachrichtigung, z.B. E-Mail. Die Benutzer*in wird eingeloggt, sobald der Login-Link in der E-Mail aufgerufen wird. Der Login-Link meldet die Benutzer*in sofort an.
```
Comment (xchs) [inline meta/richardhj/contao-email-token-login/de.yml]:
```suggestion
        Das Anmeldeformular fragt nur nach dem Benutzernamen (oder der E-Mail-Adresse). Die Benutzer*in sendet das Formular ab und erhält eine Benachrichtigung, z. B. E-Mail. Die Benutzer*in wird eingeloggt, sobald der Login-Link in der E-Mail aufgerufen wird. Der Login-Link meldet die Benutzer*in sofort an.
```

File: meta/richardhj/isotope-klarna-checkout/en.yml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+en:
+    title: Klarna Checkout v3 for Isotope eCommerce
+    description: >
+        This extensions provides an entire new checkout for Isotope eCommerce. It integrates the Klarna checkout iFrame into your shop.
+
+        The Klarna checkout will use the configuration of your Isotope shop. In addition, the extension provides two new frontend modules. You can find the documentation on GitHub.
```
Comment (xchs) [inline meta/richardhj/isotope-klarna-checkout/en.yml]:
```suggestion
        The Klarna checkout will use the configuration of your Isotope shop. In addition, the extension provides two new front end modules. You can find the documentation on GitHub.
```

**Comment by xchs** [comment]
The metadata linter fails because of the spelling of "Go". Could you add that to the whitelist, please?

**Comment by richardhj** [comment]
> The metadata linter fails because of the spelling of "Go". Could you add that to the whitelist, please?

Actually, I added "Newsletter2Go" to the dictionary, but the linter splits the compound, and thus wants to validate "Go"

**Comment by xchs** [comment]
> but the linter splits the compound, and thus wants to validate "Go"

Yeah, apparently.

File: meta/richardhj/contao-newsletter2go-sync/de.yml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+de:
+    title: Newsletter2Go-Synchronisation
+    description: >
+        Diese Erweiterung ermöglicht die Synchronisation von Contao-Mitgliedern mit Newsletter2Go.
+
+        Nach der Installation: 1) Konfigurere einen Newsletter2Go-Zugang, 2) weise den Zugang Backend-Nutzern oder in den Systemeinstellungen zu, 3) aktiviere die Synchronisation bei der Mitgliedergruppe.
```
Comment (xchs) [inline meta/richardhj/contao-newsletter2go-sync/de.yml]: Typo: Konfigurere --> Konfiguriere

**Comment by richardhj** [comment]
@fritzmg is currently blocking merging 😆

**Comment by xchs** [comment]
I see :)

**Pull request merged**